debian/changelog

```diff
@@ -1,3 +1,15 @@
+cuttlefish-common (0.9.10) UNRELEASED; urgency=medium
+
+  [ Tristan Muntsinger ]
+  * Add in required dependencies for arm64 buster
+  * Remove superfluous dependencies for cuttlefish-common
+  * Add in net-tools dependency
```

Comment:
Is this part of this PR?

Reply:
No, this is catching up as
https://github.com/google/android-cuttlefish/pull/32/files
and
https://github.com/google/android-cuttlefish/pull/31/files
did not update the changelog.